search_engine_parser/core/engines/google.py

```diff
@@ -28,6 +28,9 @@ def get_params(self, query=None, offset=None, page=None, **kwargs):
         params["start"] = (page-1) * 10
         params["q"] = query
         params["gbv"] = 1
+        # additional parameters will be considered
+        for key, val in kwargs.items():
```

Comment:
What about cases where `kwargs` contain other variables that may affect the search unknowingly. Simply adding all kwargs can be hazardous in future.

I think for best case you could do a
```python
if kwargs.get("hl"):
    params["hl"] = kwargs.get("hl")
```

This would create reproducible bugs as we will know the kwargs been used to generate the urls

Reply:
Yes it's definitely better